save-demo/src/main/kotlin/com/saveourtool/save/demo/entity/Dependency.kt

```diff
@@ -0,0 +1,22 @@
+package com.saveourtool.save.demo.entity
+
+import com.saveourtool.save.spring.entity.BaseEntity
+import javax.persistence.Entity
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+
+/**
+ * @property coreTool dependant tool
+ * @property dependencyTool the tool that [coreTool] depends on
+ */
+@Entity
+@Table(name = "dependency")
```

Comment:
This annotation is not needed here. It is needed when the class name is not equal to the table name.

Reply:
Oh thanks, will remove it

Reply:
Removed useless `Table` annotations